sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## 7.5.0-beta.1 (Unreleased)
 
 ### Features Added
+- Move to service API version 2021-05
+- Added support for specifying the maximum message size for entities in Premium namespaces.
```

Comment:
nit: could this reference `quoted` concrete types similar to past changelog entries?

Reply:
done